draft-ppm-protocol.md

```diff
@@ -409,8 +415,14 @@ struct {
 
 ## Task Configuration {#task-configuration}
 
-Prior to the start of execution of the protocol, each participant must agree on
-the configuration for each task. A task is uniquely identified by its task ID:
+A task involves evaluation of a specific Verifiable Distributed Aggregation
+Function (VDAF, see [VDAF]) over a set of client inputs. The VDAF determines the
+type of input measurements and the type of the output of the aggregation
+function. It also determines how many aggregators are involved (this document
+currently supports just two) and how many rounds of communication are required
+to verify an input's validity.  Prior to the start of execution of the protocol,
+each participant must agree on the configuration for each task. A task is
```

Comment:
Is this an appropriate place to discuss the PPM meta-protocol that would capture Prio's query randomness exchange, or is that a later PR?

Reply:
This comment is solved by having the outputs of the VDAF setup algorithm be part of the task parameters.